src/ray/gcs/gcs_server/test/gcs_server_test_util.h

```diff
@@ -321,6 +321,10 @@ struct GcsServerMocker {
       drain_raylet_callbacks.push_back(callback);
     };
 
+    void IsLocalWorkerDead(
```

Comment:
This class should be named as `Fake`, not `Mock`...

Reply:
This PR is already big. Let's do these cleanups in a separate PR.